plugins/modules/postgresql_db.py

```diff
@@ -312,8 +317,24 @@ def db_exists(cursor, db):
     return cursor.rowcount == 1
 
 
-def db_delete(cursor, db):
+def db_dropconns(cursor, db):
+    if cursor.connection.server_version >= 90200:
+        """ Drop DB connections in Postgres 9.2 and above """
+        query = ("SELECT pg_terminate_backend(pg_stat_activity.pid) FROM pg_stat_activity "
+                      "WHERE pg_stat_activity.datname=%(db)s AND pid <> pg_backend_pid()")
+    else:
+        """ Drop DB connections in Postgres 9.1 and below """
```

Comment:
```suggestion
        """ Drop DB connections in Postgres 9.1 and below """
```
A random thought but should we support very old unsupported versions?

Reply:
Maybe in this case, i would leave this because it doesn't look complex.

Reply:
I agree the code is simple enough to support that old version.